app/controllers/admin/routes.py

```diff
@@ -40,6 +42,61 @@
 from app.controllers.admin import admin_bp
 from app.logic.spreadsheet import createSpreadsheet
 
+@admin_bp.route('/event/<eventId>/cohortStatus', methods=['GET'])
```

Comment:
Could we potentially move this into the edit/create event route where it is rendering the page already? That way we don't have to do an extra request?

Reply:
Done